docs/tutorials/reproducing_pober_2014.ipynb

```diff
@@ -0,0 +1,721 @@
+{
```

Comment:
<p>This isn't quite right.  The code at the time used aipy cal files, which contained a longitude and latitude.  For this paper, the array was actually in Green Bank!  </p><p> </p><p>'loc': ('38:25:59.24',&nbsp;'-79:51:02.1'), # Green Bank, WV</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/steven-murray/21cmSense/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>